asterius/src/Asterius/Builtins.hs

```diff
@@ -117,19 +117,14 @@ rtsAsteriusModule opts =
           , ( "__asterius_func"
```

Comment:
`__asterius_func` used to be a global variable when multi-threads was absent. Now, if one thread calls an async import and suspends, and another thread does the same, `__asterius_func` of the first thread can get overwritten right? So this should also be made a thread-local thing like `__asterius_regs` and `__asterius_ret`